.ci/scripts/functional-test.sh

```diff
@@ -29,6 +29,6 @@ BEAT_VERSION=${4:-"${BASE_VERSION}"}
 rm -rf outputs || true
 mkdir -p outputs
 
-REPORT="$(pwd)/outputs/TEST-${SUITE}"
+REPORT="$(pwd)/outputs/TEST-${GOARCH}-${SUITE}"
```

Comment:
`GOARCH` is missing in the variable definitions, is that intended?

Reply:
Mmm, this is set by the pipeline, within a `withEnv` block. Are you concerned about using this script outside the CI? If so, you are right, as the GOARCH is not set. Do you suggest passing it as a positional parameter instead? I think it would be a simple change.

Reply:
https://github.com/elastic/e2e-testing/pull/707/files#diff-e2a25c72a2cfde08a03b6735e6672600d998351ea4265e6b6146386aeb709e3bR17 uses that approach, that's the reason for my previous comment

Reply:
Ok, will proceed as in that commit. Thanks!!